spring-boot-project/spring-boot-docs/src/docs/asciidoc/web/spring-security.adoc

```diff
@@ -224,6 +224,37 @@ Again, the same properties are applicable for both servlet and reactive applicat
 Alternatively, you can define your own `OpaqueTokenIntrospector` bean for servlet applications or a `ReactiveOpaqueTokenIntrospector` for reactive applications.
 
 
+=== Audience Validation Support in OAuth2 Resource Server
+
+The OAuth2 resource server now supports validating audience (aud) claims in JSON Web Tokens (JWTs) issued by an authorization server. 
+This feature is added to complement the existing validation of issuer (iss) claims.
+
+To enable audience validation, set the spring.security.oauth2.resourceserver.jwt.audience property in your Spring Boot application 
```

Comment:
I think the suggested additions are more verbose than they need to be and adding just this paragraph and perhaps the example that follows it would be sufficient.

The configuration property should be referenced using the `configprop:` syntax:

```
configprop:spring.security.oauth2.resourceserver.jwt.audiences[]
```

This will ensure that it's a valid configuration property.

Reply:
> The configuration property should be referenced using the `configprop:` syntax:
> 
> ```
> configprop:spring.security.oauth2.resourceserver.jwt.audiences[]
> ```

I have documented this. Waiting for your review.

Reply:
Thanks for the updates, @bikash30851. The changes are now in the right place but the content isn't quite what I meant. The guidance about the syntax was for you. I was describing how the documentation should be written. It wasn't intended as content to be added to the documentation. Sorry that wasn't clear. If you look elsewhere in the file that you're editing, you'll see other usages of the `configprop:` syntax. Your changes should match those.

Reply:
@wilkinsona  Apologies! I'll update the documentation to match the existing usage of the configprop: syntax. Thanks for the clarification!